types/blueimp-load-image/index.d.ts

```diff
@@ -149,7 +167,24 @@ interface LoadImage {
     // Replaces the image head of a JPEG blob with the given one
     replaceHead: ReplaceHead;
 
-    writeExifData: (buffer: ArrayBuffer | Uint8Array, data: loadImage.MetaData, id: number | string, value: loadImage.ExifTagValue) => ArrayBuffer | Uint8Array;
+    writeExifData: (
+        buffer: ArrayBuffer | Uint8Array,
+        data: loadImage.WriteExifData,
+        id: number | string,
+        value: loadImage.ExifTagValue,
+    ) => ArrayBuffer | Uint8Array;
+
+    scale: Scale;
+
+    // Internal functions, undocumented
+    requiresMetaData: (options: loadImage.LoadImageOptions) => boolean;
+    fetchBlob: (url: string, callback: () => void) => void;
+    transform: (img: unknown, options: unknown, callback: () => void, file: unknown, data: unknown) => void;
+    global: Window;
+    readFile: unknown;
+    isInstanceOf: unknown;
+    createObjectURL: (blob: Blob) => string | false;
+    revokeObjectURL: (url: string) => void | false;
```

Comment:
Mostly LGTM.

One question - if these are internal and undocumented, is there a good reason to include them in the typings?